mmv1/third_party/terraform/services/compute/data_source_google_compute_instance.go.erb

```diff
@@ -59,7 +59,7 @@ func dataSourceGoogleComputeInstanceRead(d *schema.ResourceData, meta interface{
 	if err != nil {
 		return err
 	}
-	if err := d.Set("network_inferface", networkInterfaces); err != nil {
+	if err := d.Set("network_interface", networkInterfaces); err != nil {
```

Comment:
looks like a typo slipped through from: https://github.com/GoogleCloudPlatform/magic-modules/pull/11276/files#diff-8256fc32389e6b295f4714811edd92aee6dbfee67b386847ac0f528d1301aef2L62

No idea why it is just now showing up in VCR